docs/installing-solidity.rst

```diff
@@ -73,7 +73,7 @@ Docker
 ======
 
 Docker images of Solidity builds are available using the ``solc`` image from the ``ethereum`` organization.
-Use the ``stable`` tag for the latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+Use the ``stable`` tag for the latest released version, and ``nightly`` for potentially unstable changes in the development branch.
```

Comment:
This is not a typo - the branch is named ``develop``.

Reply:
yes you are right. Thanks for that! sorry my mistake.

Reply:
@r0qs: Thanks for your suggestion, this is really helpful!
I've modified now the sentence according to your suggestion.